### PR TITLE
Task update 

### PR DESCRIPTION
**Pull request by Astarsys**
- Added descriptions, examples, table parameters.

### CODE REVIEW

File: docs/backend-api/resources/field_service/form/index.md [hunk 1]
```diff
@@ -167,15 +167,17 @@ Gets form by an id.
...
+      },
       "state": "uploaded",
       "download_url": "https://static.navixy.com/file/dl/1/0/1g/01gw2j5q7nm4r92dytolzd6koxy9e38v.png/lala.jpg"
     }]
 }
 ``` 
 
 * `value` - A [form object](#form object).
-* `files` - list of [form_file objects](#form_file object). Files used in values of this form. Can be null or empty.
+* `files` - list of [form_file objects](#form file object). Files used in values of this form. Can be null or empty.
```
Comment (fenik17) [inline docs/backend-api/resources/field_service/form/index.md:180]: Is it link really works?

File: docs/backend-api/resources/field_service/task/form/index.md [hunk 2]
```diff
@@ -199,89 +267,117 @@ Returns descriptions of forms, created on the basis of specified form template.
...
-metadata object:
-
-```json
-{
- "orientation":  <int, image exif orientation>,
-}
-```
-
-For **form** object description, see [task/form/](#form-object-structure).
+* `value` - [form object](../../form/index.md#form object), or null if no form attached.
```
Comment (fenik17) [inline docs/backend-api/resources/field_service/task/form/index.md:365]: I think, a hyphen is required in the link: `...#form-object`.

File: docs/backend-api/resources/field_service/task/checkpoint.md [hunk 3]
```diff
@@ -94,67 +144,167 @@ Delete checkpoint with the specified id.
...
+          "origin": "imported",
+          "tags": [1, 2],
+          "type": "checkpoint",
+          "form": <form_object>
+    }
 }
 ```
 
-where **checkpoint** described [here](#checkpoint).
+* `value` - `checkpoint` object described [here](#checkpoint object).
```
Comment (fenik17) [inline docs/backend-api/resources/field_service/task/checkpoint.md:274]: Whitespace in the link.

**Review by fenik17 — CHANGES_REQUESTED** [review]

File: docs/backend-api/resources/field_service/task/checkpoint.md (hunk 3 above)
Comment (Astarsys) [reply docs/backend-api/resources/field_service/task/checkpoint.md]: links with spaces really work with whitespaces, but I changed spaces to `-`

**Comment by Astarsys** [comment]
links with spaces really work with whitespaces, but I changed spaces to -

**Review by fenik17 — APPROVED** [review]
@Astarsys Cool. Thanks!

**Pull request merged**